app/build.gradle

```diff
@@ -16,6 +16,11 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+        debug{
+            buildConfigField 'String', 'USER_SAVED', '"user_saved"'
```

Comment:
@SimonMaha 'USER_SAVED' and 'USER_EXIST' should not be here. Only build variables goes here.